docs/clone-sst.md

```diff
@@ -0,0 +1,163 @@
+
+## Limitations
+
+Clone limitations are described in [Clone plugin limitations](clone-plugin-limitations.html)
+
+## Key features
+
+| Feature                 | Description                                                 |
+|-------------------------|-------------------------------------------------------------|
+| Efficient data transfer | The clone plugin transfers data at the file level, reducing overhead. |
+| Consistency            | Ensures data consistency between the donor and joiner nodes. |
+| Minimal Downtime       | Reduces the time required for node synchronization.         |
+| Native Integration     | Fully integrated into MySQL, eliminating the need for external tools. |
+
+
+## Prerequisites
+
+The requirements for enabling SST transfers with the Clone plugin are as follows:
+
+* Percona XtraDB Cluster (PXC) version 8.0.41 or later
+
+* Sufficient disk space and network bandwidth for data transfer
+
+* Properly configured PXC cluster with at least one donor node
+
+* NetCat package installed
+
+* Port 4444 is used by the State Snapshot Transfer (SST) process in Percona XtraDB Cluster for data transfer between nodes
+
+## Best practices
+
+The following best practices are for using SST with the Clone plugin:
+
+| Recommendation           | Description                                                                                  |
+|-------------------------------|--------------------------------------------------------------------------------------------------|
+| Choose a Suitable Donor       | Select a donor node with low load and sufficient resources to avoid performance degradation.     |
+| Monitor Resources             | Monitor CPU, memory, and disk usage during the SST process.                                      |
+| Test in Staging               | Test the SST process in a staging environment before deploying it in production.                 |
+
+## Process outline
+
+A high-level outline of the process:
+
+<div style="text-align: center;">
+  <img src="./_static/clone-sst-process.png" alt="Clone SST process" width="200" />
+</div>
+
+## Enable the method
+
+To enable SSL for Percona XtraDB Cluster, ensure that certificates are manually or automatically placed in a specific location on both the donor and joiner servers. Avoid using the data directory because the clone process modifies this directory.
+
+In the configuration file (`my.cnf`), set the `wsrep_sst_allowed_methods` variable to include the `clone` method on both the donor and joiner servers.
+
+The `wsrep_sst_allowed_methods` and `wsrep-sst-method` variables are read-only and cannot be changed at runtime; they must be set before starting the server. The variables must be set in the configuration file before starting the server.
```

Comment:
Here we start saying about `wsrep-sst-method`, but there was nothing about it before.
Let's add before this paragraph, that in my.cnf the user has to set 'wsrep-sst-method=clone'. This variable affects joiner (joiner requests SST that uses this method)

Reply:
+1